File: matlab/scaleFeatureMatrix.m
function [scaledFeatureMatrix] = scaleFeatureMatrix(featureMatrix,typ)
%Scales feature matrix.
%    :param featureMatrix: feature matrix (shape: [n,w])
%    :param typ: name/tag of sclaer to be used.
%    :return scaledFeatureMatrix: scaled feature matrix (shape: [n,w])
if typ == "none"
    scaledFeatureMatrix = featureMatrix;
else
    scaledFeatureMatrix = normalize(featureMatrix,2,typ);
end
end

File: r/scale_feature_matrix.R
scale_feature_matrix <- function(feat_matrix,normMethod) {
#  Scales feature matrix.
#  :param feat_matrix: feature matrix (shape: [n,w])
#  :param normMethod: name/tag of scaler to be used.
#  :return: scaled feature matrix (shape: [n,w])
  if (normMethod == "none") {
    return(feat_matrix)
  } else {
    return(t(scale(t(feat_matrix))))
  }
}